docs/pages/database-access/guides/aws-cassandra-keyspaces.mdx

```diff
@@ -3,11 +3,11 @@ title: Database Access with Amazon Keyspaces (Apache Cassandra)
 description: How to configure Teleport database access with Amazon Keyspaces (Apache Cassandra)
 ---
 
-(!docs/pages/includes/database-access/db-introduction.mdx  dbType="Amazon Keyspaces (Apache Cassandra)" dbConfigure="Amazon Keyspaces database with IAM authentication" dbName="Amazon Keyspaces" !)
+(!docs/pages/includes/database-access/db-introduction.mdx dbType="AWS Keyspaces (Apache Cassandra)" dbConfigure="with IAM authentication"!)
 
 ## How it works
 
-(!docs/pages/includes/database-access/how-it-works/iam.mdx db="Amazon Keyspaces" cloud="AWS"!)
```

Comment:
why not keep the `Amazon` part?

Reply:
I thought it was redundant because of cloud=AWS, but I see that the snippet is fine with both. I'll revert it to keep the `Amazon` bit (and the others that I changed `Amazon -> AWS`)